### PR TITLE
consumable screens added

### PR DESCRIPTION
**Pull request by PrashanthReddyPR**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6288,11 +6288,14 @@
   dependencies:
     "minimist" "^1.2.6"
 
+<<<<<<< Updated upstream
```
Comment (Gorantasathish) [inline yarn.lock:6291]: @PrashanthReddyPR  why you pushed this lock file and it has conflicts

File: src/components/Sidebar/index.js [hunk 2]
```diff
@@ -47,6 +47,10 @@ const Sidebar = () => {
     if( user_role == 'Admin'){
       setSidebar(adminSidedata);      
     }
+    /* if( user_role == 'Admin'){
```
Comment (Gorantasathish) [inline src/components/Sidebar/index.js:50]: @PrashanthReddyPR  why you commented this lines?

File: src/components/FixedRequest/index.js [hunk 3]
```diff
@@ -57,21 +57,16 @@ const FixedRequest = () => {
...
-
-
             });
-
             console.log("result", result)
             setRequestData(result);
             setFilterValue("")
           
 
+           // setApproval(result);
```
Comment (Gorantasathish) [inline src/components/FixedRequest/index.js:66]: hope we can remove this commented lines right @PrashanthReddyPR

File: src/components/CentralConsumableAssetIssue/index.js [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+                toast.error('Something went wrong please try again');
+            }
+        }
+    }
+
+    useEffect(() => {  
+        const fetchAsset = async () => {           
+            let result = await API_Auth.getAsset();                    
+            setAssets(result);
+        /*   let vendorList = await API_Auth.getVendors();
```
Comment (Gorantasathish) [inline src/components/CentralConsumableAssetIssue/index.js:94]: remove commented code if not required

File: src/API_Auth.js [hunk 5]
```diff
@@ -480,13 +489,23 @@ import {
         ).json();
         return result;
         },
-        updateItemToAsset:async (assetID,toStoreID,voucher,price,tax,mfd,expirydate,supplier,comments,batchNo,itemUID) =>{
-          let url = UPDATE_ITEM_TO_ASSET_API_URL+"assetID="+assetID
+ /*        updateItemToAsset:async (assetID,toStoreID,voucher,price,tax,mfd,expirydate,supplier,comments,batchNo,itemUID) =>{
+          let url = UPDATE_ITEM_TO_ASSET_API_URL+"assetID="+assetID */
```
Comment (Gorantasathish) [inline src/API_Auth.js:493]: is it merged corred?

**Review by Gorantasathish — CHANGES_REQUESTED** [review]

File: src/components/Sidebar/index.js (hunk 2 above)
Comment (PrashanthReddyPR) [reply src/components/Sidebar/index.js:50]: i didnt do any changes in this file

File: src/components/FixedRequest/index.js (hunk 3 above)
Comment (PrashanthReddyPR) [reply src/components/FixedRequest/index.js:66]: yes ill remove

File: src/components/CentralConsumableAssetIssue/index.js (hunk 4 above)
Comment (PrashanthReddyPR) [reply src/components/CentralConsumableAssetIssue/index.js:94]: ok

File: src/API_Auth.js (hunk 5 above)
Comment (PrashanthReddyPR) [reply src/API_Auth.js:493]: i didnt get this

Comment (PrashanthReddyPR) [reply src/API_Auth.js:493]: i didnt understand this

File: src/components/Sidebar/index.js (hunk 2 above)
Comment (Gorantasathish) [reply src/components/Sidebar/index.js:50]: yeah you missed something while resolving conflicts

**Review by Gorantasathish — APPROVED** [review]

**Pull request merged**